CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # web-ledger-client ChangeLog
 
+### 3.4.2 - TBD
```

Comment:
I believe this needs to be a breaking change since the error signatures will not be the same for apisauce and httpClient, right? And so anywhere there are tests in dependent libraries that make assertions about errors from here will probably need to be updated..

Reply:
now a major release: https://github.com/digitalbazaar/web-ledger-client/pull/25/commits/b8030d31ae9ee0de2ce49b19a92ce2d8a1bfca9d